app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/react/DeployStatus.tsx

```diff
@@ -64,7 +64,7 @@ export class DeployStatus extends React.Component<IExecutionDetailsSectionProps,
   private manifestIdentifier(manifest: IStageManifest) {
     const kind = manifest.kind.toLowerCase();
     // manifest.metadata.namespace doesn't exist if it's a namespace being deployed
-    const namespace = (manifest.metadata.namespace || '').toLowerCase();
+    const namespace = (manifest.metadata.namespace || '_').toLowerCase();
```

Comment:
not strictly necessary to be part of identifier but a visual indicator when debugging